src/brevitas/nn/quant_conv.py

```diff
@@ -175,8 +176,9 @@ def per_elem_ops(self):
     @property
     def output_channel_dim(self):
         if self.transposed:
-            raise RuntimeError("Transposed kernels not supported")
```

Comment:
@Giuseppe5, is there a reason why this RuntimeError was raised? Or is Tim's fix for this OK?

Reply:
I believe this to be correct